hack/e2e/ensure_cmctl.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+USR_LOCAL_BIN="/usr/local/bin/"
```

Comment:
Unnecessary trailing `/` 

```suggestion
USR_LOCAL_BIN="/usr/local/bin"
```